packages/broccoli-eyeglass/src/broccoli_sass_compiler.ts

```diff
@@ -627,26 +627,33 @@ export default class BroccoliSassCompiler extends BroccoliPlugin {
    * @return hash object of the file data
    **/
   hashForFile(absolutePath: string): string {
-    let cacheKey = `hashForFile(${absolutePath})`;
-    let cachedHash = this.buildCache.get(cacheKey);
-    if (cachedHash) {
-      return cachedHash as string;
+    return this.fileKey(absolutePath);
+  }
+
+  /* compute a key for a file that will change if the file has changed. */
+  fileKey(file: string, isRealPath = false): string {
+    let cachedKeyKey = `fileKey(${file})`;
+    let cachedKey = this.buildCache.get(cachedKeyKey) as string;
+    if (cachedKey) { return cachedKey; }
+    let stat = statSync(file);
+    let key;
+    if (!isRealPath && stat.isSymbolicLink) {
```

Comment:
Also, I believe `statSync` [by default follows symlinks](http://man7.org/linux/man-pages/man2/fstat.2.html), given that when would the result of `statSync(xyz).isSymbolicLink() === true`?

Reply:
Evidently never. I've removed the symbolic link handling. 🎉